.github/workflows/scripts/kinds/verify-kinds.go

```diff
@@ -182,47 +175,32 @@ func die(errs ...error) {
 
 // verifyKind verifies that stable kinds are not updated once published (new schemas
 // can be added but existing ones cannot be updated)
-func verifyKind(registry *kindRegistry, kind kindsys.Kind, name string, category string, latestRegistryDir string) error {
+func verifyKind(registry *kindRegistry, maturity string, name string, category string, latestRegistryDir string) error {
 	oldKindString, err := registry.getPublishedKind(name, category, latestRegistryDir)
 	if err != nil {
 		return err
 	}
 
-	var oldKind kindsys.Kind
-	if oldKindString != "" {
-		switch category {
-		case "core":
-			oldKind, err = loadCoreKind(name, oldKindString)
-		case "composable":
-			oldKind, err = loadComposableKind(name, oldKindString)
-		default:
-			return fmt.Errorf("kind can only be core or composable")
-		}
-	}
-	if err != nil {
-		return err
-	}
+	ctx := cuecontext.New()
+	oldKind := ctx.CompileBytes(oldKindString)
 
 	// Kind is new - no need to compare it
-	if oldKind == nil {
+	if oldKind.Validate() != nil {
 		return nil
 	}
 
+	// Do we still need this??
```

Comment:
Naively, I'm tempted to say that we don't. Cog doesn't use this, and from what I remember the maturity concept was never really used to its full extent.